api/proto/config/storage.proto

```diff
@@ -106,3 +106,48 @@ message Drive {
   // A value of 0 will indicate that no resizing is required
   int64 maxsizebytes = 10;
 }
+
+
+message ContentTree {
+  UUIDandVersion uuidandversion = 1;
+  string dsId  = 2; // effectively pointer/key into dsConfigs
+
+  string URL = 3;
+  Format iformat = 4;
+
+  // the following is only used for individual blobs, if this message references a group of blobs
+  // e.g. in the case of OCI -- this information is expected to be provided by a top level blob
+  // that this message points to via its URL
+  string sha256 = 5;
+  uint64 sizeBytes = 6; // used for capping resource consumption eve for OCI & BLOB_TREE
+  SignatureInfo siginfo = 7;
+}
+
+enum VolumeAccessProtocols {
+  VAP_NONE = 0;
+  VAP_9P = 1;
+  VAP_NFS = 2;
+  VAP_SCSI = 3;
+  VAP_VIRTIO = 4;
+}
+
```

Comment:
Let's not define these until we have a plan to implement them. Otherwise they are likely to appear in the UI and then we have to remove them from the UI.

Reply:
removed everything else and only kept VAP_9P for now. Also, this is EVE API, the UI API is different and the controller provides a different(user-friendly) abstraction/API to the UI.

Reply:
OK